crates/nu-protocol/src/config.rs

```diff
@@ -42,7 +42,9 @@ impl Hooks {
             pre_prompt: None,
             pre_execution: None,
             env_change: None,
-            display_output: None,
+            display_output: Some(Value::test_string(
```

Comment:
the doc of `Value::test_string` says
> Note: Only use this for test data, *not* live data, as it will point into unknown source
> when used in errors.

we might not want to use that here, right? :thinking:

Reply:
@amtoine don't have a choice, as we don't have any span information at that location.

Reply:
could we use `Value::string` and give a `Span::unknown`?
that might be less confusing than seeing the doc of `Value::test_string`, comparing it to the location of this call and seeing that it shouldn't be used :yum:

Reply:
something like
```rust
Value::string(
    "if (term size).columns >= 100 { table -e } else { table }",
    Value::Span::unknown(),
)
```

Reply:
That's the same thing. I can change it, but it doesn't really add value.

Reply:
:pray: